shared/login/register/code-page/index.render.desktop.js

```diff
@@ -37,24 +35,28 @@ export default class CodePageRender extends Component {
         </p>
         <Icon type='computer-bw-m' style={{marginTop: 28}} />
 
-        <Text type='Body' style={styles.paperkey}>{this.props.textCode}</Text>
+        <Text type='Body' style={stylesPaperkey}>{this.props.textCode}</Text>
       </Container>
     )
   }
 
-  renderCode () {
+  _otherIsPhone () {
+    return [codePageDeviceRoleExistingPhone, codePageDeviceRoleNewPhone].indexOf(this.props.otherDeviceRole) !== -1
+  }
+
+  _renderCode () {
     const qr = {
       background: `url("${this.props.qrCode}")`
     }
 
     return (
       <Container
-        style={styles.container}
+        style={stylesContainer}
         onBack={this.props.onBack}>
         <Text style={{marginTop: 38, marginBottom: 11}} type='Header'>Scan this QR code</Text>
-        {subTitle()}
```

Comment:
stateless components can be treated like components. we never need to use this form